src/components/CarouselUnequip/index.tsx

```diff
@@ -0,0 +1,24 @@
+import Carousel, { CarouselItem } from "../Carousel";
+import CardSpell from "../CardSpell";
+import { UniqueCardName, Power } from "../../types";
+import { getPowerByName } from "../../data-accessor";
+
+export interface Props {
+  cards: UniqueCardName[];
+  onClose: () => void;
+  onSelectItem: (id: UniqueCardName) => void;
+}
+
+export default function UnequipCarousel({ cards, onClose, onSelectItem }: Props) {
```

Comment:
nit: I feel like we could just make a generic carousel at this point that takes a list of card names, buttonText, and renderNode

Reply:
agreed